lib/handbook/addon/docs/components/tags-widget/controller.ts

```diff
@@ -0,0 +1,17 @@
+import { action } from '@ember-decorators/object';
+import Controller from '@ember/controller';
+
+export default class TagsWidgetController extends Controller {
+    // required arguments
```

Comment:
I don't know if this comment adds anything here since we're only defining required arguments and could maybe even be a little confusing?